io-package.json

```diff
@@ -190,21 +190,25 @@
 		"store_quarters" : false,
 		"store_years" : false,
 		"store_period" : false,
-		"keep_days": "7",
-		"keep_weeks": "4",
-		"keep_quarters" : "2",
-		"keep_months" : "4",
-		"keep_years" : "1",
-		"unit_price_power" : "0.27",
-		"unit_price_power_night" : "0.24",
-		"unit_price_power_delivery" : "0.17",
-		"basic_price_power" : "15",
-        "unit_price_heatpump": "0.24",
-        "unit_price_heatpump_night": "0.22",
-		"unit_price_water" : "0.07",
-		"basic_price_water" : "22",
-        "unit_price_oil": "0.12",
-		"basic_price_oil" : "31"
+		"keep_days": 7,
+		"keep_weeks": 4,
+		"keep_quarters" : 2,
+		"keep_months" : 4,
+		"keep_years" : 1,
+		"unit_price_power" : 0.27,
+		"unit_price_power_night" : 0.24,
+		"unit_price_power_delivery" : 0.17,
+		"basic_price_power" : 15,
+        "unit_price_heatpump": 0.24,
+        "unit_price_heatpump_night": 0.22,
+        "basic_price_heatpump": 31,
+		"unit_price_water" : 0.07,
+		"basic_price_water" : 22,
+        "unit_price_oil": 0.12,
+        "basic_price_oil" : 31,
+        "unit_price_gas": 0.12,
+        "basic_price_gas" : 31,
```

Comment:
I added a few missing ones, you might want to check the default values.

Reply:
thank you, figuring out what happened here as it looks in the repro but somehow not into the code